crates/derive/src/combine_api.rs

```diff
@@ -147,8 +162,11 @@ impl cgo::Service for API {
             _ => Err(Error::InvalidState),
         };
 
-        let diff = mem_stats.current() - initial;
-        tracing::trace!(mem_initial = ?initial, mem_changes = ?diff, "mem changes");
+        // Get the difference in memory allocations for this invocation, and use it to update our
+        // running tally.
+        let diff = (mem_stats.current() - initial).total_allocated();
+        self.allocated_bytes += diff;
+        tracing::trace!(allocated_bytes = ?self.allocated_bytes, allocated_bytes_delta = ?diff, "mem usage");
```

Comment:
This log seems verbose, even for trace level.

(And if we're going to leave it in, it should probably live within the cgo service implementation).